### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by TRNEEDANAME**
Add the french translation of the "Prototype Armoury" mod

### CODE REVIEW

File: PrototypeArmoury/Localization/FRA/Items/XcomGame.fra [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[TLE_KevlarArmor X2ArmorTemplate]
+FriendlyName="Armure récupérée".
+FriendlyNamePlural="Armurse récupérée".
```
Comment (boundir) [inline PrototypeArmoury/Localization/FRA/Items/XcomGame.fra]: Armur~~s~~e

File: PrototypeArmoury/Localization/FRA/Items/XcomGame.fra [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[TLE_KevlarArmor X2ArmorTemplate]
+FriendlyName="Armure récupérée".
+FriendlyNamePlural="Armurse récupérée".
+BriefSummary="Cette armure de base consiste en une protection improvisée à partir de caches d'armes. Elle offre une protection minimale contre les armes ADVENT et aliens."
+TacticalText="<Bullet/> Tous les soldats reçoivent une armure en Kevlar dans leur équipement de base.<br />La recherche peut nous donner accès à une meilleure armure."
+
+[TLE_PlatedArmor X2ArmorTemplate]
+FriendlyName="Prototype d'armue renforcée"
+FriendlyNamePlural="Prototypes d'armues renforcée"
```
Comment (boundir) [inline PrototypeArmoury/Localization/FRA/Items/XcomGame.fra]: Armure is wrong on line 8 and 9 -> `armue`

File: PrototypeArmoury/Localization/FRA/Items/XcomGame.fra [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+[TLE_PlatedArmor X2ArmorTemplate]
+FriendlyName="Prototype d'armue renforcée"
+FriendlyNamePlural="Prototypes d'armues renforcée"
+BriefSummary="Utilisant des plaques récupérées d'armures d'ADVENT, cette combinaison offre une protection accrue et une capacité d'inventaire supplémentaire à nos soldats."
+TacticalText="<Bullet/> Armure de poids moyen basée sur des récupérations d'ADVENT.\n<Bullet/> En tant qu'armure de classe moyenne, elle offre un emplacement pour un objet d'inventaire supplémentaire."
+
+[TLE_PoweredArmor X2ArmorTemplate]
+FriendlyName="Armure motorisée expérimentale".
+FriendlyNamePlural="Armure motorisées expérimentale".
```
Comment (boundir) [inline PrototypeArmoury/Localization/FRA/Items/XcomGame.fra]: Plural of armure and experimentale is missing

File: PrototypeArmoury/Localization/FRA/Items/XcomGame.fra [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+BriefSummary="Notre nouvelle maîtrise de la technologie Elerium des extraterrestres nous a permis de mettre au point l'armure de combat ultime. L'armure alimentée à l'Elerium offre une capacité de survie et une polyvalence sans précédent dans la lutte contre des ennemis extraterrestres de plus en plus puissants."
+TacticalText="<Bullet/> Une armure de classe moyenne de premier ordre est disponible.<br />L'armure électrique dispose de 2 emplacements pour des objets dans l'inventaire."
+
+[SparkBit_CV X2GremlinTemplate]
+BriefSummary="Le BIT est une variation de notre drone GREMLIN, avec les sous-systèmes medikit et electroshock remplacés par un montage d'armes lourdes."
```
Comment (boundir) [inline PrototypeArmoury/Localization/FRA/Items/XcomGame.fra]: électrochoc

File: PrototypeArmoury/Localization/FRA/POIs/XcomGame.fra [hunk 5]
```diff
@@ -0,0 +1,49 @@
+[POI_Prototype_T2 X2PointOfInterestTemplate]
+DisplayNames[0]="SOS de la Résistance".
+
+CompletedSummaries[0]="Lors d'un balayage dans cette zone, nous avons détecté un signal SOS provenant d'une structure cachée dans les ruines de la ville. Il semble que certains scientifiques aient fait défection à ADVENT et aient continué leurs recherches en se cachant. Nous ne savons pas ce qui leur est arrivé, mais sous une poutre de soutien effondrée, nous avons trouvé un prototype fonctionnel qui semble dépasser tout ce que nous avons encore développé."
+
+POIImages[0]="img:///UILibrary_StrategyImages.X2StrategyMap.POI_OvergrownCity"
+
+[Récompense_PrototypeT2 X2RewardTemplate]
```
Comment (boundir) [inline PrototypeArmoury/Localization/FRA/POIs/XcomGame.fra]: The header should not be translated

**Comment by TRNEEDANAME** [comment]
All of the issues should now be fixed. Thanks to @boundir for pointing them out

File: PrototypeArmoury/Localization/FRA/POIs/XcomGame.fra [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+DisplayName="Prototype avancé"
+
+[POI_Sidegrade_T1 X2PointOfInterestTemplate]
+DisplayNames[0]="Bunker de l'ancien monde".
+
+CompletedSummaries[0]="La résistance locale nous a signalé l'existence d'un bunker de l'ancien monde caché dans les montagnes. Les scanners du Vengeur l'ont facilement localisé, mais malheureusement les pièces les plus intérieures du bunker ont été éliminées par une frappe directe d'une arme à plasma orbitale. Cependant, il y avait plusieurs pièces secondaires qui étaient suffisamment éloignées pour être épargnées, et dans l'une d'entre elles, nous avons trouvé une armurerie avec quelques armes anciennes intactes."
+
+POIImages[0]="img:///UILibrary_StrategyImages.X2StrategyMap.POI_RadioTower"
+
+[Récompense_SidegradeT1 X2RewardTemplate]
```
Comment (boundir) [inline PrototypeArmoury/Localization/FRA/POIs/XcomGame.fra]: One header left here 😉

Comment (TRNEEDANAME) [reply PrototypeArmoury/Localization/FRA/POIs/XcomGame.fra]: Fixed with new commit

**Pull request merged**